frontend/src/lib/uPlotLib/plugins/tooltipPlugin.ts

```diff
@@ -67,7 +68,13 @@ const generateTooltipContent = (
 					unit = '',
 				} = seriesList[index - 1] || {};
 
-				const value = data[index][idx];
+				// eslint-disable-next-line no-nested-ternary
+				const value = stackBarChart
+					? index + 1 < data.length
```

Comment:
Can right this cleaner.

Reply:
moved this to a util function and added it as if else rather than ternary